L1Trigger/L1THGCalUtilities/plugins/ntuples/HGCalTriggerNtupleHGCDigis.cc

```diff
@@ -77,7 +78,17 @@ DEFINE_EDM_PLUGIN(HGCalTriggerNtupleFactory, HGCalTriggerNtupleHGCDigis, "HGCalT
 
 HGCalTriggerNtupleHGCDigis::HGCalTriggerNtupleHGCDigis(const edm::ParameterSet& conf) : HGCalTriggerNtupleBase(conf) {
   is_Simhit_comp_ = conf.getParameter<bool>("isSimhitComp");
-  digiBXselect_ = conf.getParameter<std::vector<int>>("digiBXselect");
+  digiBXselect_ = conf.getParameter<std::vector<unsigned int>>("digiBXselect");
+
+  if (*max_element(digiBXselect_.begin(), digiBXselect_.end()) >= kDigiSize_) {
```

Comment:
In general I prefer to explicitly use `std::`  (`std::max_element()`) to make clear it is from the standard library